### ISSUE TITLE
In Sketcher link actions menu keeps popping up when selecting points

### ISSUE THREAD

**Issue by mccarthyam** [FreeCAD/FreeCAD#16549]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Problem description

I am trying in sketcher to select two points as add a constraint but the link actions menu keeps popping up preventing me from adding the constraint 

See attached screen video of the issue

https://github.com/user-attachments/assets/5d73059c-b581-4576-8011-a828277f4f04

### Full version info

```shell
1.0.0RC1
```

### Subproject(s) affected?

None

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by luzpaz** [comment]
@mccarthyam full About info, please

**Comment by mccarthyam** [comment]
Please find about infor
![image](https://github.com/user-attachments/assets/91f33797-abc8-4fca-9e5b-410c5c498444)

**Comment by Roy-043** [comment]
I suspect this is [Mouse navigation](https://wiki.freecad.org/Mouse_navigation) related. Please check if the issue occurs with the "CAD navigation" option.

**Comment by mccarthyam** [comment]
Tried it with the CAD navigation option and the problem does not re-occur

**Comment by Roy-043** [comment]
Which mouse navigation do you use?
Do you hold down the Ctrl key (which is not necessary) when selecting elements in sketch edit mode?

**Comment by mccarthyam** [comment]
I used the gesture navigation. I changed to TinkerCad and the issue has not re-occured

**Comment by luzpaz** [comment]
@Rexbas another gesture navigation issue, have you seen this before ?

**Comment by Rexbas** [comment]
Yes, I have seen similar behavior in other navigation styles and I have fixed them some time ago. I will see if I can also fix this too.

**Comment by Rexbas** [comment]
I cannot reproduce this. @mccarthyam, could you please describe in detail what you are doing. For example: press LMB, move mouse, release LMB. And are you using a mouse or a laptop touchpad?

**Comment by luzpaz** [comment]
@Rexbas can you add the [About](https://wiki.freecad.org/About) info of the version you tested? TIA

@mccarthyam can you test on latest weekly build and report back? (https://github.com/FreeCAD/FreeCAD-Bundle/releases/tag/weekly-builds) TIA

**Comment by Rexbas** [comment]
OS: Fedora Linux 40 (KDE Plasma) (KDE/plasma/xcb)
Word size of FreeCAD: 64-bit
Version: 1.1.0dev.38991 (Git)
Build type: Release
Branch: main
Hash: ef2156d0f6d116b1642a5531a2d44fc6b7d17042
Python 3.11.0, Qt 5.15.8, Coin 4.0.3, Vtk 9.3.0, OCC 7.7.2
Locale: English/United States (en_US)
Stylesheet/Theme/QtStyle: OpenDark.qss/OpenDark/Fusion
Installed mods: 
  * OpenTheme 2024.9.1
  * OpticsWorkbench 1.0.23
